Steps/Steps.module.less

```diff
@@ -16,14 +16,14 @@
 			font-weight: @sand-steps-step-number-font-weight;
 		}
 
-		&.dots {
-			font-size: @sand-steps-step-dot-font-size;
+		&.highlightCurrent {
+			font-size: @sand-steps-step-highlightCurrent-font-size;
```

Comment:
Do we need this overriding? If we set it to `18` in size prop.

Reply:
I think we need to override font-size. The circle needs to fit inside the 18px div according to the GUI guide, but if the font-size remains 18px, the circle will be much smaller than the GUI guide.